File: app/src/main/java/team/triplog/activity/SplashActivity.java
package team.triplog.activity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.view.WindowManager;
import android.view.animation.AlphaAnimation;
import android.view.animation.Animation;

import androidx.appcompat.app.AppCompatActivity;

import team.triplog.R;

public class SplashActivity extends AppCompatActivity {
    long DURATION = 2500L;
    View viewLogo;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getWindow().setFlags(
                WindowManager.LayoutParams.FLAG_FULLSCREEN,
                WindowManager.LayoutParams.FLAG_FULLSCREEN
        );

        setContentView(R.layout.activity_splash);

        init();
    }

    private void init() {
        viewLogo = findViewById(R.id.view_logo);
        setAnimation();
    }

    private void setAnimation() {
        AlphaAnimation anim = new AlphaAnimation(0.0f, 1.0f);
        anim.setDuration(DURATION);
        anim.setStartOffset(0);
        anim.setRepeatMode(Animation.REVERSE);
        anim.setRepeatCount(Animation.ABSOLUTE);
        anim.setAnimationListener(new Animation.AnimationListener() {
            @Override
            public void onAnimationEnd(Animation animation) {
                Intent intent = new Intent(getApplicationContext(), MainActivity.class);
                startActivity(intent);
                finish();
            }

            @Override
            public void onAnimationStart(Animation animation) {
            }

            @Override
            public void onAnimationRepeat(Animation animation) {
            }
        });

        viewLogo.startAnimation(anim);
    }
}

File: app/src/main/java/team/triplog/presentation/splash/SplashActivity.kt
package team.triplog.presentation.splash

import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity
import team.triplog.presentation.main.activity.startMainActivity

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        Handler().postDelayed({ moveNext() }, 500)
    }

    private fun moveNext() {
        // TODO : SharedPreference 로 로그인 관리 필요
        // TODO : Login 된 UserId 가 있을 경우, 분기처리 필요
        startMainActivity()
        finish()
    }
}